exporters/failure/collector_base.py

```diff
@@ -8,6 +8,9 @@
 
 import pelorus
 
+# TBD_1: CI needs to create failures on the fly to enable this
```

Comment:
What is `TBD_1`?

Reply:
**T**o**B**e**D**one with number. Currently I can not enable this part, because we have pre-stored failures for various APIs (github, jira, ...) but I want to deliver this fix asap, so I commented out part which should be enabled. Currently it doesn't really matter as Prometheus will drop the old metrics anyway, however to be consistent with deploytime this part should be enabled in the future and we have also somewhere idea of creating failures during e2e tests.

Reply:
Changed.